gdnative-core/src/core_types/error.rs

```diff
@@ -76,7 +76,7 @@ impl GodotError {
 impl std::fmt::Display for GodotError {
     #[inline]
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        write!(f, "Godot API error: {:?}", self)
+        write!(f, "godot api error: {:?}", self)
```

Comment:
"Godot API" should still be capitalized. "Godot" is a proper noun, and API is an abbreviation. A similar case can be seen in the examples given in the guidelines: "invalid UTF-8 sequence of {} bytes from index {}"

Reply:
I went ahead and reverted that change.